doc/computing/parallelism.rst

```diff
@@ -10,38 +10,42 @@ Parallelism, resource management, and configuration
 Parallelism
 -----------
 
-Some scikit-learn estimators and utilities can parallelize costly operations
-using multiple CPU cores, thanks to the following components:
+Some scikit-learn estimators and utilities parallelize costly operations
+using multiple CPU cores.
 
-- via the `joblib <https://joblib.readthedocs.io/en/latest/>`_ library. In
-  this case the number of threads or processes can be controlled with the
-  ``n_jobs`` parameter.
-- via OpenMP, used in C or Cython code.
+This can be either done:
 
-In addition, some of the numpy routines that are used internally by
-scikit-learn may also be parallelized if numpy is installed with specific
-numerical libraries such as MKL, OpenBLAS, or BLIS.
+- with higher-level parallelism via `joblib <https://joblib.readthedocs.io/en/latest/>`_.
+  In this case, the number of threads or processes can be controlled with the
+  ``n_jobs`` parameter.
+- with lower-level parallelism via OpenMP, used in C or Cython code.
+  In this case, parallelism is always done using threads and specifying
+  ``n_jobs`` *has no effect*. Implementations relying on this parallelism are generally
+  more performant by several orders of magnitude.
```

Comment:
More performant than what? A group of monkeys with calculators? :D

Reply:
Perfect timing for using this emoji: :see_no_evil: 

Is this clearer? :upside_down_face: 
```suggestion
  more performant than joblib-based implementations by several orders of magnitude.
```